projects/ngx-d3timeline/src/lib/axis/resources-axis/resources-axis.component.ts

```diff
@@ -27,8 +27,9 @@ import { AxisService } from '../axis.service';
   styles: [
     `
       .resources-axis-group {
-        text-anchor: middle;
+        dominant-baseline: central;
```

Comment:
This will vary depending on the orientation once the `cx` and `cy` are removed from `getTranslation` should probably be part of the `Tick` class

Reply:
text anchor I had in the Tick class and will re-add. I reverted some changes and brought it back here by accident.

dominantBaseLine I had in there too but read a few places where people said to avoid it and use cx/cy/transform instead. 
https://stackoverflow.com/questions/35565063/svg-dominant-baseline-not-working-in-safari

Thats why I put cx/cy into getTranslation, ultimately we are just saying where do we posiition the tick, shouldn't way say that in one way rather than two that need to be combined?

Reply:
I'd pref to use the `dominant-baseline` css rather than setting it in the transform. 

Either way this css should be removed